.env

```diff
@@ -16,6 +16,8 @@ if [ ! -d "venv" ]; then
      if ! hash pyvenv 2>/dev/null; then
         function pyvenv()
         {
+            if hash pyvenv-3.7 2>/dev/null; then
+                pyvenv-3.7 $@
             if hash pyvenv-3.6 2>/dev/null; then
```

Comment:
This could be __elif hash pyvenv-3.6__ instead of __if hash pyvenv-3.6__ but it is minor thing.